service/src/main/java/gov/nasa/pds/api/registry/model/Pds4ProductBusinessObject.java

```diff
@@ -87,8 +90,16 @@ public int setResponse(HitIterator hits, Summary summary, List<String> fields) {
       uniqueProperties
           .addAll(ProductBusinessObject.getFilteredProperties(kvp, fields, null).keySet());
 
-      Pds4Product prod = Pds4ProductFactory.createProduct(hits.getCurrentId(), kvp, this.isJSON);
-      list.add(prod);
+      try {
+        Pds4Product prod = Pds4ProductFactory.createProduct(hits.getCurrentId(), kvp, this.isJSON);
+        list.add(prod);
+      } catch (Throwable t) {
+        String lidvid = "unknown";
+        if (kvp.containsKey("lidvid")) {
+          lidvid = kvp.get("lidvid").toString();
```

Comment:
Suggest: remove "CRITICAL: ", as this suggests a greater-than-ERROR log significance and is confusing.

Should this even be an ERROR?  Bad data seems like a candidate for WARNING